### DISCUSSION TITLE
Commercial use question

### DISCUSSION THREAD

**Discussion by happybuny005** [dashingsoft/pyarmor#1235] [Q&A] [answered]
Hello all!

I'd like to use Pyarmor to obfuscate my program's code, but I have a question regarding commercial use. My program is completely free to download and use... but I do accept donations through PayPal (from the download page only. My program doesn't contain any donation links). Does this still count as commercial use? If so, does a basic license grant me access to commercial? Thank you in advance for any answers provided.

**Comment by jondy** [comment]
Why do you obfuscate your code if your program is totally free to download and use?

I think it still count as commercial use.

**Reply by happybuny005** [reply]
It is free to download and use but I'd still like for it to be "closed-source". The Python release is a port to my Windows application which is also closed-source. 

One more question if you don't mind me asking, does the basic license work? In the [License file](https://github.com/dashingsoft/pyarmor/blob/master/LICENSE) (here on github), it states that the basic license allows for commercial, but on the [readthedocs](https://pyarmor.readthedocs.io/en/stable/licenses.html#id5) for pyarmor it states on the license page that you need pro for commercial. Can you please clarify which one it is? Again, thank you so much for the reply!

**Reply by jondy** [reply, marked as answer]
Sorry, basic license allows for commercial, it's missed in readthedocs.

Thanks, I'll fixed in soon.

**Reply by happybuny005** [reply]
Ah, thank you so much for the clarification! I really do appreciate it!
